.github/workflows/check-links.yaml

```diff
@@ -40,8 +40,7 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - name: Install dependencies
-        run: |
-          pip install -r userdocs/requirements.txt
+        run: make install-site-deps
```

Comment:
Looking at https://github.com/weaveworks/eksctl/pull/2157/checks?check_run_id=658968162 it seems this pulls in a bunch of Go dependencies too?

Reply:
wow good spot. I have taken a closer look, it is due to two attributes _eksctl_version_ and _godeps_cmd_ in Makefile.common and Makefile.

Please refer the commit (used for testing only) https://github.com/weaveworks/eksctl/pull/2157/commits/c82d5794225edd7689a168d856e45e1af283a689

Here is the respective github action run
https://github.com/weaveworks/eksctl/pull/2157/checks?check_run_id=665342849

Reply:
Mh. Looks like the Go deps are still being pulled. So yeah. Not sure if we really need this here? No big deal in any case.

BTW we had the same conversation here already: https://github.com/weaveworks/eksctl/pull/2062#discussion_r412518621

Reply:
Oh sorry, I reverted the testing commit, and github didn't keep the run if there was forced push. Should have capture the screen shot of github action before :)

Reply:
i like the idea of keeping these commands all under `make`.
at the moment, it's not _really_ a problem to pull in the go deps, am I understanding that right?
if it becomes a problem, I would fix it in the `Makefile`.

Reply:
You are right. If required, we can change the Makefile to avoid running any go command running outside of make target :)

Reply:
yeah, no problem